components/home/Dashboard.tsx

```diff
@@ -83,44 +62,143 @@ export default function Dashboard() {
       </View>
     </View>
   );
+};
+interface Transaction {
+  amount: string;
+  category: string;
+}
+const calculateCO2E = (transactionHistory = [], amountPerKg = 0.41) => {
```

Comment:
The 0.41 factor is actually only if we want to say how much carbon dioxide emissions saved by the user saving compost - but the text we're showing is saying how many kilos of garbage are prevented. Let's remove the factor of 0.41 here.